src/Surfnet/AzureMfa/Infrastructure/Controller/DefaultController.php

```diff
@@ -116,4 +127,20 @@ public function authenticationAction(Request $request)
             'NameID' => $nameId ?: 'unknown',
         ], $response);
     }
+
+    /**
+     * @Route("/acs", name="app_identity_acs")
+     */
+    public function acsAction(Request $request)
+    {
+        try {
+            $this->azureMfaService->handleResponseFromAdfs($request);
```

Comment:
For good measure, you might want to add the final line of code to finalize this registration request.

`$this->registrationService->replyToServiceProvider();`